### DISCUSSION TITLE
Request for help writing docs

### DISCUSSION THREAD

**Discussion by marcelfolaron** [Leantime/leantime#1100] [General]
Hi Everyone,

We are in desperate need for some technical writers who would like to update our documentation. Everything is written in markdown on our docs repository: https://github.com/Leantime/docs

If you are interested feel free to create a PR. 
Some topics that would be great to cover:

- Nginx proxy install
- Install on ubuntu via apache or nginx
- Updating user docs on how to use Leantime with updated screenshots

**Comment by globule** [comment]
OK for install on Apache after my problem is solved ! ;) #1106

**Reply by globule** [reply]
We can start from this https://www.osradar.com/leantime-project-management-ubuntu-server/

**Comment by broskees** [comment]
@marcelfolaron Have you considered implementing [PHPDoc](https://www.phpdoc.org/)? Now that we have the `custom` directory and the events/filters system coming out in 2.3, it might be more useful then before.
